CHANGELOG-unreleased.md

```diff
@@ -10,6 +10,8 @@ the released changes.
 ## Unreleased
 ### Changed
 ### Added
+- Added WaveX model as DelayComponent with wave amplitudes as fitted parameters
```

Comment:
Please also mention the changes you made in the Wave model as a different entry (maybe in changed or fixed section)